src/components/CommentForm.jsx

```diff
@@ -1,26 +1,27 @@
 import React, { useState } from 'react';
 import { postComment } from '../utils/api';
+import ErrorMessage from './ErrorMessage';
 import '../App.css';
 
 function CommentForm({ articleId, onCommentPosted }) {
     const [commentBody, setCommentBody] = useState('');
-    const [errorMessage, setErrorMessage] = useState('');
+    const [error, setError] = useState('');
 
     const handleSubmit = async (e) => {
         e.preventDefault();
 
         if (!commentBody.trim()) {
-            setErrorMessage('Comment cannot be empty.');
+            setError('Comment cannot be empty.');
```

Comment:
I think this error message could be more descriptive for the user i.e. how to remedy the problem